aiida/cmdline/commands/cmd_verdi.py

```diff
@@ -12,12 +12,38 @@
 from __future__ import print_function
 from __future__ import absolute_import
 
+import difflib
 import click
 
 from aiida.cmdline.params import options, types
 
 
-@click.group(context_settings={'help_option_names': ['-h', '--help']})
+class MostSimilarCommandGroup(click.Group):
+    """Overloads the get_command to display a list of possible command
+    candidates if the command could not be found with an exact match."""
+
+    def get_command(self, ctx, cmd_name):
+        cmd = click.Group.get_command(self, ctx, cmd_name)
+
+        # return the exact match
+        if cmd is not None:
+            return cmd
+
+        # we might get better results with the Levenshtein distance
+        # or more advanced methods implemented in FuzzyWuzzy or similar libs,
+        # but this is an easy win for now
+        matches = difflib.get_close_matches(cmd_name, self.list_commands(ctx), cutoff=0.5)
+
+        ctx.fail(
```

Comment:
if what you type is "too far off", matches will be empty.
In that case, one should rather print something like `No similar commands found` (or nothing).

Perhaps also add a quick test for that.

Reply:
done, let's see what the CI thinks